plugins/node/plugin.zsh

```diff
@@ -0,0 +1,39 @@
+# Color definitions
+GEOMETRY_COLOR_NODE_NPM_VERSION=${GEOMETRY_COLOR_NODE_NPM_VERSION:-black}
+
+# Symbol definitions
+GEOMETRY_SYMBOL_NODE_NPM_VERSION=${GEOMETRY_SYMBOL_NODE_NPM_VERSION:-"⬡"}
+GEOMETRY_NODE_NPM_VERSION=$(prompt_geometry_colorize $GEOMETRY_COLOR_NODE_NPM_VERSION $GEOMETRY_SYMBOL_NODE_NPM_VERSION) 
+
+
+geometry_prompt_node_setup(){} 
+
+geometry_prompt_node_render(){
+    local -a files
+    files=($(ls -a ${PWD} | egrep -e ".nvmrc|yarn.lock"))
+
+    if [[ -n $files[1] ]]; then
+        case "$files[1]" in 
+            ".nvmrc")
+                local NODE_VERSION=$(nvm version "$(cat $files[1])")
+                if [[ "$NODE_VERSION" != "N/A" ]]; then
+                    nvm use 2> /dev/null
```

Comment:
I'm not sure how I feel about this. If I understand it correctly, and please correct me if I'm wrong, but it makes changes to the environment instead of being idempotent and just reading the version. It assumes the user wants to use the version specified on `.nvmrc` which is not something we should feel comfortable about enforcing.

Does anyone else have takes on this?